src/server/services/controller.ts

```diff
@@ -91,15 +92,16 @@ const CMS_MAINTENANCE_NOTIFICATIONS = callPublicService(
 // Protected services
```

Comment:
Ziet er misschien whacky uit zo maar ik had het gesorteerd, alleen `PARKEREN` is nieuw toegevoegd